clusterloader2/testing/node-throughput/windows_override.yaml

```diff
@@ -1,8 +1,8 @@
 WINDOWS_NODE_TEST: true
 POD_COUNT: 80
 POD_THROUGHPUT: 0.03
-CONTAINER_IMAGE: gcr.io/gke-release/pause-win:1.1.0
+CONTAINER_IMAGE: gcr.io/gke-release/pause-win:1.2.0
 OPERATION_TIMEOUT: 90m
 POD_STARTUP_LATENCY_THRESHOLD: 60m
-WMI_EXPORTER_URL: "https://github.com/martinlindhe/wmi_exporter/releases/download/v0.10.1/wmi_exporter-0.10.1-amd64.msi"
+WMI_EXPORTER_URL: "https://github.com/YangLu1031/wmi_exporter/releases/download/v0.10.3-alpha/wmi_exporter.exe"
```

Comment:
nit: I'm assuming that at some point an official version will be relased. So how about dropping a TODO to replace with an official version once it will appear?